src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -13,7 +13,11 @@
             [showResult]="showResults && latestRatedResult != undefined"
         >
         </jhi-exercise-details-student-actions>
-        <a *ngIf="plagiarismCaseId" class="btn btn-danger btn-sm me-2" [routerLink]="['/courses', courseId, 'plagiarism-cases', plagiarismCaseId]">
+        <a
+            *ngIf="plagiarismCaseInfo && plagiarismCaseInfo.verdict !== PlagiarismVerdict.NO_PLAGIARISM"
```

Comment:
```suggestion
            *ngIf="plagiarismCaseInfo?.verdict !== PlagiarismVerdict.NO_PLAGIARISM"
```

Reply:
We need to disable the button when `plagiarismCaseInfo` is `undefined` so the suggested version won't cover it unfortunately.